metals/src/main/scala/scala/meta/internal/metals/ClientCommands.scala

```diff
@@ -34,6 +34,8 @@ object ClientCommands {
        |export interface DoctorOutput {
        |  /** Metals Doctor title */
        |  title: string;
+       |  /** Version of the doctor json format, 0 if empty  */
```

Comment:
@ckipp01 I also fixed the docs here so that we can link to it from the reworked doctor paragraph

Reply:
Ahh good call 👍🏼